### PR TITLE
Add thumbs count

### PR DESCRIPTION
**Pull request by nicolehli**
- Added action and reducer code to have thumbs up and down counter count.
- Commented out Heart icon code because it is a stretch goal and we don't need it right now
- Also added a banana picture as placeholder image

SOLVED: Credits to Erica for figuring I was missing mapStateToProps in card.jsx. 
Need a second pair of eyes on `Card.jsx` onThumbsUpPressed function on line 28. I am trying to add a function to add thumbs up/down that will increase a counter.

### CODE REVIEW

File: imports/ui/components/Card.jsx [hunk 1]
```diff
@@ -23,16 +25,56 @@ class CardComponent extends Component {
     this.setState({ isFav: !this.state.isFav });
   };
 
+  onThumbsUpPressed = () => {
+    let item = this.state.data;
+    item.count = item.count + 1;
+    this.props.changeCount(item);
+
+    // TODO BUG: How come redux is change but this component doesn't rerender?
+    console.log(this.props);
```
Comment (nicolehli) [inline imports/ui/components/Card.jsx]: So changeCount has updated redux, but this component doesn't rerender 🤷‍♀

**Comment by YaoLiu93** [comment]
A newly added item doesn't have a number associated. Hitting one of the thumbs up/down button renders NaN.

Maybe for future consideration: Add a timeout or cookie to store if the user has clicked the button recently. Otherwise, users can just spam the rating.

File: imports/ui/reducers/CardReducer.js [hunk 2]
```diff
@@ -11,6 +12,7 @@ const initialState = {
       item: "Bananas",
       price: "$0.99",
       uuid: "41048778-017e-44d6-9496-6b60a7c81d8f",
+      count: 10,
```
Comment (YaoLiu93) [inline imports/ui/reducers/CardReducer.js]: Please pull in master's changes and update the add item caller to add an initial count to the `item` payload.

Comment (YaoLiu93) [reply imports/ui/reducers/CardReducer.js]: Can we also rename to "rating" instead of count?

**Pull request merged**